src/manifold/src/impl.cpp

```diff
@@ -902,4 +903,57 @@ SparseIndices Manifold::Impl::VertexCollisionsZ(
   else
     return collider_.Collisions<false, false>(vertsIn);
 }
+/*
+ * Returns the minimum gap between two manifolds. Returns a float between
+ * 0 and searchLength.
+ */
+float Manifold::Impl::MinGap(const Manifold::Impl& other,
+                             float searchLength) const {
+  ZoneScoped;
+  auto getSortedExpandedFaceBoxMorton =
+      [searchLength](const Manifold::Impl& impl) {
+        Vec<Box> faceBox;
+        Vec<uint32_t> faceMorton;
+
+        impl.GetFaceBoxMorton(faceBox, faceMorton);
+
+        transform(autoPolicy(faceBox.size()), faceBox.begin(), faceBox.end(),
+                  faceBox.begin(), [searchLength](const Box& box) {
+                    return Box(box.min - glm::vec3(searchLength),
+                               box.max + glm::vec3(searchLength));
+                  });
+
+        // impl.SortFaceBoxMorton(faceBox, faceMorton);
+        return std::pair{faceBox, faceMorton};
+      };
+
+  auto [faceBox, faceMorton] = getSortedExpandedFaceBoxMorton(*this);
+  auto [faceBoxOther, faceMortonOther] = getSortedExpandedFaceBoxMorton(other);
```

Comment:
If you expand them both by the search length, I believe you're actually searching twice the search length.

Reply:
Oh, you're right!